.github/workflows/snow-cr-notification.js

```diff
@@ -0,0 +1,44 @@
+const { slackNotification, getLocalConfigs } = require('./helpers.js');
+
+async function main({ github, context, transaction_id, change_id, planned_start_time, planned_end_time } = {}) {
```

Comment:
This _should_ work, I'm not certain though. Would only assign the values from getLocalConfigs when github/context are otherwise undefined
```suggestion
async function main({ github = getLocalConfigs().github, context = getLocalConfigs().context } = {}) {
```

Reply:
made the adjustments to have all needed params with defaults if not supplied for local execution. Transaction_id, change_id, etc. will be assigned by default the environment variables otherwise you could send in whatever you would like to test it locally.